packages/core/src/server.ts

```diff
@@ -1773,11 +1865,28 @@ export class Controller extends API {
 
         await this.storage.save(auth);
 
+        this.auditor.log("auth", auth.id, account.id, "updated", { accountId: auth.account });
+
         this.log("account.deleteKeyStoreEntry", {
             keyStoreEntry: { id: entry.id },
         });
     }
 
+    async itemAuditorLog({ itemId, auditChangeType, metadata }: ItemAuditorLogParams): Promise<void> {
+        const { account } = this._requireAuth();
+
+        // TODO: Remove this
+        console.log("======== server.itemAuditorLog");
+        console.log(JSON.stringify({ itemId, auditChangeType, metadata }, null, 2));
```

Comment:
So, I don't know what I'm missing in the deserialization, but it's always an empty object (it looks like a properly serialized object being sent in the request). Help @MaKleSoft